dags/vz_afd_ems_import.py

```diff
@@ -0,0 +1,83 @@
+# stuff to make the airflow, 1Password integration work
+import os
+import pendulum
+from airflow.decorators import dag, task
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development')   # our current environment from ['production', 'development']
+
+ENVIRONMENT = {
+    "ENVIRONMENT": DEPLOYMENT_ENVIRONMENT,
+    "OP_API_TOKEN": os.getenv("OP_API_TOKEN"),
+    "OP_CONNECT": os.getenv("OP_CONNECT"),
+    "OP_VAULT_ID": os.getenv("OP_VAULT_ID"),
+}
+
+# EMS DAG
+
+# define the parameters of the DAG
+@dag(
+    dag_id="vz-ems-import",
+    description="A DAG which imports EMS data into the Vision Zero database.",
+    schedule="0 7 * * *",
+    start_date=pendulum.datetime(2023, 1, 1, tz="UTC"),
+    catchup=False,
+    tags=["repo:atd-vz-data", "vision-zero", "ems", "import"],
+)
+
+def etl_ems_import():
+    import docker
+
+    @task()
+    def run_ems_import_in_docker():
+        client = docker.from_env()
+        docker_image = "atddocker/vz-afd-ems-import:latest"
+        # pull at run time; remember to publish a amd64 image to docker hub
```

Comment:
@johnclary For sure. I think it's a good point that if it's worth putting a note in a comment that folks should rebuild the image if they change it, then it's worth building CI to do it. [This](https://hub.docker.com/repository/docker/atddocker/vz-afd-ems-import/general) will get CI'd.

To drive this point home on your behalf, I 100% had to rebuild the CRIS image because I had change it since last build and it tripped me up during deployment today. My natural workflows just don't yet have this concept of dockerhub being a place to store images the way we do, so I'm too quick to forget about it and related CI.

Reply:
CI worked out really well. thanks for this suggestion!